charmhub/client.go

```diff
@@ -129,14 +94,85 @@ func NewClient(config Config) (*Client, error) {
 		return nil, errors.Annotate(err, "constructing info path")
 	}
 
-	httpClient := DefaultHTTPTransport()
-	apiRequester := NewAPIRequester(httpClient)
+	findPath, err := base.Join("find")
+	if err != nil {
+		return nil, errors.Annotate(err, "constructing find path")
+	}
+
+	var (
```

Comment:
Nit: why not save 2 lines and just use `:=`?

Reply:
It was doing something else before, I simplified it now.